docs/specifications.md

```diff
@@ -8,7 +8,7 @@ order: 100
 The following versions are available, and the specific version in use will depend on your configuration.
 
 - PHP Version: 7.4, 8.0, 8.1, 8.2  see [Altis compatibility chart](docs://guides/updating-php/#altis-compatibility-chart)
-- MySQL Version: 5.7 (compatible; using AWS Aurora)
+- MySQL Version: 8.0 (compatible; using AWS Aurora)
```

Comment:
As suggested in the daily, we should remove mentions of vendor-specific service names like AWS Aurora.

Reply:
This was intentionally here as Aurora differs from regular MySQL